frontend/app/views/info/supporterUSA.scala.html

```diff
@@ -0,0 +1,82 @@
+@(pageInfo: model.PageInfo, pageImages: Seq[model.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token)
+
+@import com.gu.membership.salesforce.Tier
+@import model.Benefits
+@import configuration.Videos
+
+@main("Supporters", pageInfo=pageInfo) {
+
+    @* ===== About Supporters ===== *@
+    @for(img <- pageImages.find(_.name.contains("intro"))) {
+        <div class="page-slice l-constrained">
+            @fragments.promos.promoPrimary(title="Become a Guardian Supporter", img=img, isLead=true) {
+                <div class="text-feature">
+                    <p>Supporters keep our journalism fearless, open and free from&nbsp;interference.</p>
+                </div>
+                @fragments.tier.packagePromoUSASupporter(isReversed=false)
+            }
+        </div>
+    }
+
+    @* ===== Support open, independent journalism ===== *@
+    <div id="supporter-video" class="page-slice page-slice--slim l-constrained">
+        @fragments.promos.promoVideoSecondary(
+            Html("Support open, independent journalism"),
```

Comment:
can the use of `Html` be removed here? all clients of this template use only a `String`.